bin/terraform-apply.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# Convenience script for running terraform apply for the specified module and configuration name.
+# The configuration name is used to determine which .tfvars file to use for the -var-file
+# option of terraform apply. Additional arguments to terraform apply can also be passed in.
```

Comment:
Yeah that sounds nice. Only thing is that autocomplete and variables is a bit annoying with Makefiles, where it's like `make target ARGS="some string of args"` rather than `make target some args`, which is why I gave the common make commands specific make targets so that ohmyzsh can autocomplete when you type `make infra-<tab>`.

What do you think about doing that in a follow up ticket?

And what do you think about the instructions for running terraform directly in https://github.com/navapbc/template-infra/pull/242/files#diff-546e28d202b614e4cbc4c27f03c9bda9a502d143e5cdb167a87d47feb545c67dR51 ? Any ideas for improving those?